src/Simulation/Common/IQubitManager.cs

```diff
@@ -3,41 +3,37 @@
 
 namespace Microsoft.Quantum.Simulation.Common
 {
-    using Microsoft.Quantum.Simulation.Core;
     using System.Collections.Generic;
+    using Microsoft.Quantum.Simulation.Core;
 
     public interface IQubitManager
     {
```

Comment:
This also contain breaking changes. We'll need to update the minor (I believe Sarah did some other breaking changes recently, so it's probably a good opportunity)

Reply:
Yes.